[RemoteControl] The framework 'Microsoft.AspNetCore.App', version '2.2.0' was not found.

##  Current behavior

Running on a recent visual studio installation without .NET Core 2.2 installed raises the following error:

```
[INFO] Uno Remote Control initialized (3.0.0-dev)
[INFO] Loaded the Uno.UI Remote Control service (3.0.0-dev).
[DEBUG]
[ERROR] It was not possible to find any compatible framework version
[ERROR] The framework 'Microsoft.AspNetCore.App', version '2.2.0' was not found.
[ERROR]   - The following frameworks were found:
[ERROR]       2.1.18 at [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
[ERROR]       3.1.4 at [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
[ERROR]
[ERROR] You can resolve the problem by installing the specified framework and/or SDK.
```

## Expected behavior

RemoteControl installation works properly.

## Workaround

Installing the 2.2.0 hosting bundle https://dotnet.microsoft.com/download/dotnet-core/thank-you/runtime-aspnetcore-2.2.0-windows-hosting-bundle-installer works around the issue.

## Environment

<!-- For bug reports Check one or more of the following options with "x" -->

Nuget Package: any

--- a/.vsts-ci.yml
+++ b/.vsts-ci.yml
@@ -15,7 +15,7 @@ pr:
 resources:
   containers:
   - container: nv-bionic-wasm
-    image: unoplatform/wasm-build:2.0
+    image: unoplatform/wasm-build:2.1.1
 
 variables:
   windowsVMImage: 'windows-2019'

--- a/build/Uno.WinUI.RemoteControl.nuspec
+++ b/build/Uno.WinUI.RemoteControl.nuspec
@@ -85,7 +85,7 @@
 	<file src="..\src\Uno.UI.RemoteControl\bin\Release\xamarinmac20\Uno.UI.RemoteControl.pdb" target="lib\xamarinmac20" />
 
 	<!-- Remote Control host and integration -->
-    <file src="..\src\Uno.UI.RemoteControl.Host\bin\Release\netcoreapp2.2\*.*" target="tools\rc\host" />
+    <file src="..\src\Uno.UI.RemoteControl.Host\bin\Release\netcoreapp3.1\*.*" target="tools\rc\host" />
     <file src="..\src\Uno.UI.RemoteControl.VS\bin\Release\net461\*.dll" target="tools\rc" />
     <file src="..\src\Uno.UI.RemoteControl.VS\bin\Release\net461\*.pdb" target="tools\rc" />
     <file src="..\src\Uno.UI.RemoteControl.VS\bin\Release\net461\Newtonsoft.Json.dll" target="tools\rc" />

--- a/src/Uno.UI.RemoteControl.Host/Uno.UI.RemoteControl.Host.csproj
+++ b/src/Uno.UI.RemoteControl.Host/Uno.UI.RemoteControl.Host.csproj
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
 		<CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
 	</PropertyGroup>
 
@@ -15,6 +15,7 @@
 	<ItemGroup>
 		<PackageReference Include="Microsoft.AspNetCore.App" />
 		<PackageReference Include="Mono.Options" Version="6.6.0.161" />
+		<PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
 		<PackageReference Include="System.Reactive" Version="4.4.1" />
 		<PackageReference Include="Unity" Version="5.11.7" />
 		<PackageReference Include="Uno.Core" Version="2.0.0" />

